src/qutipinterface.jl

```diff
@@ -0,0 +1,192 @@
+    expectation = zeros(Complex128, length(tlist)-1, length(eops))
+    j = 1
+    for (t,rho) in QuStateEvolution(H, c_ops, rho0, tlist, solver)
+        for i=1:length(eops)
+            expectation[j,i] = expectationvalue(rho, eops[i])
+        end
+        j = j + 1
+    end
+    return collect(tlist)[2:end], expectation
+end
+
+mesolve(ME::QuLindbladMasterEq, rho0::QuBase.AbstractQuMatrix, tlist, eops::Vector; solver=QuExpmV()) = mesolve(ME.hamiltonian, rho0, tlist, ME.collapse_ops, eops; solver=solver)
+
+
+@doc """
+Interface for `mcsolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.mcsolve] of QuTiP in QuDynamics
+
+`mcsolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/ME :: AbstractQuArray/QuLindbladMasterEq Types
+
+  This parameter supports Hamiltonian of the system, Lindblad Master Equation
+* psi :: AbstractQuArray
+
+  Initial state vector of the system
+* tlist
+
+  Time step array/range
+* e_ops
+
+  Operators whose expectation values are to computed in the evolved states
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
+function mcsolve(H::QuBase.AbstractQuArray, psi0::QuBase.AbstractQuArray, tlist, c_ops::Vector; ntraj=1000)
+    time = collect(tlist)[2:end]
+    qumcwfen = QuMCWFEnsemble(complex(psi0), ntraj)
+    rho = complex(psi0*psi0')
+    rhos = Array(typeof(rho), length(tlist)-1)
+    for i=1:length(tlist)-1
+        rhos[i] = complex(zeros(rho))
+    end
+    for psi0 in qumcwfen
+        j = 1
+        for (t,psi) in QuStateEvolution(H, c_ops, psi0, tlist, QuMCWF())
+            rhos[j] = rhos[j] + (psi*psi')/length(qumcwfen)/norm(psi)^2
+            j = j + 1
+        end
+    end
+    return time, rhos
+end
+
+mcsolve(ME::QuLindbladMasterEq, psi0::QuBase.AbstractQuArray, tlist; ntraj=1000) = mcsolve(ME.hamiltonian, psi0, tlist, ME.collapse_ops; ntraj=ntraj)
+
+function mcsolve(H::QuBase.AbstractQuArray, psi0::QuBase.AbstractQuArray, tlist, c_ops::Vector, eops::Vector; ntraj=1000)
+    time = collect(tlist)[2:end]
+    qumcwfen = QuMCWFEnsemble(complex(psi0), ntraj)
+    expectation = zeros(Complex128, length(tlist)-1, length(eops))
+    for psi0 in qumcwfen
+        j = 1
+        for (t,psi) in QuStateEvolution(H, c_ops, psi0, tlist, QuMCWF())
+            for i=1:length(eops)
+                expectation[j,i] = expectation[j,i] + expectationvalue(psi, eops[i])/length(qumcwfen)/norm(psi)^2
+            end
+            j = j + 1
+        end
+    end
+    return time, expectation
+end
+
+mcsolve(ME::QuLindbladMasterEq, psi0::QuBase.AbstractQuArray, tlist, eops::Vector; ntraj=1000) = mcsolve(ME.hamiltonian, psi0, tlist, ME.collapse_ops, eops; ntraj=ntraj)
+
+function eops_helper(eops, evolved_states)
```

Comment:
This could act as a test for `QuTiP` being a module. As we call `QuDynamics.QuTiP.eops_helper` in the test suite. Though we do `using QuDynamics.QuTiP` at the top of the test file `qutipinterfacetest.jl` but still I think this could act as a test for `QuTiP` being a loaded as a module.